Automattic/woocommerce-payments
# Simplify Deposits UI - Turn on new deposits overview UI by default

Fixes #6016 

#### Changes proposed in this Pull Request









This PR makes our new deposits overview UI the default UI for merchants when they visit **Payments > Overview**.

<img width="1166" alt="Screenshot 2023-04-14 at 4 28 12 pm" src="https://user-images.githubusercontent.com/2275145/231961161-c7f67c6c-ba5c-4bea-a528-fed08e5d3f61.png">

* This PR does not remove the old deposits overview UI
* To revert back to the old UI set `_wcpay_feature_simplify_deposits_ui` to `0` in wp-options table.

#### Testing instructions





1. Delete the `_wcpay_feature_simplify_deposits_ui` from wp-options table
2. Visit **Payments > Overview**
3. Confirm the new UI is used as the default
4. Set `_wcpay_feature_simplify_deposits_ui` to `0` and refresh the overview page
5. Confirm the previous deposits overview UI returns

-------------------


**Post merge**
